docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,206 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
+## Overview
+
+A `hasMany` relation denotes a one-to-many connection of a model to another
+model through referential integrity. The referential integrity is enforced by a
+foreign key constraint on the target model which usually references a primary
+key on the source model. This relation indicates that each instance of the
+declaring or source model has zero or more instances of the target model. For
+example, in an application with customers and orders, a customer can have many
+orders, as illustrated in the diagram below.
+
+![hasMany relation illustration](./imgs/hasMany-relation-example.png)
+
+The target model, **Order**, has a property, **customerId**, as the foreign key
```

Comment:
Nitpick: to give context, lead the sentence off with `The diagram shows target model ...`. Also, no commas needed here